decidim-core/app/packs/src/decidim/input_emoji.js

```diff
@@ -29,7 +32,22 @@ export default function addInputEmoji() {
       wrapper.appendChild(elem);
       wrapper.appendChild(btnContainer);
 
-      btnContainer.addEventListener("click", () => picker.togglePicker(btnContainer))
+      btnContainer.addEventListener("click", handlerPicker);
+
+      document.addEventListener("characterCounter",function(event) {
```

Comment:
I believe here the `elem` variable is referring to the input element (if I'm reading this correct) so you can attach the event listener to the `elem` instead after doing the above change in the character counter:

```js
elem.addEventListener("characterCounter", (event) => {...});
```

Please also use the ES6(+) arrow function here instead of a wrapped function.

Within the event, yo don't have to get the target element from the event. You can just work directly with the `btnContainer` removing the if statement.

Reply:
I didn't know it worked that way thanks a lot !